sites/friday/src/migration/from_platformsh.md

```diff
@@ -0,0 +1,271 @@
+To do so, commit your files and deploy your application using the
+Upsun CLI:
+  $ git add .
+  $ git commit -m 'Add Upsun configuration files'
+  $ upsun project:set-remote
+  $ upsun push
+```
+{{< note >}}
+as this command generate the minimum configuration to install the detected stack, you would probably need to compare your previous Yaml section from your `.platform.app.yaml` (or `.platform/applications.yaml`) file to your new `.upsun/config.yaml` file and report possible missing settings, like your additional command lines in your `build`/`deploy` hooks section.
+</br>When it’s done, commit and push your changes.
+{{< /note >}}
+
+## Defined app and services resources
+Within Platform.sh Yaml configuration files, it is possible to define resources of your app and services by using such settings.
+Example for your applications:
+```yaml
+# .platform.app.yaml
+name: app
+type: nodejs:16
+# How many resources to devote to the app. Defaults to AUTO in production environments.
+# More information: https://docs.platform.sh/create-apps/app-reference.html#sizes
+size: L
+# Fine-tune allocated resources
+# https://docs.platform.sh/create-apps/flexible-resources.html#performance-profiles
+resources:
+  base_memory: 1024
+  memory_ratio: 1024
+# The size of the persistent disk of the application (in MB). Minimum value is 128
+disk: 1024
+```
+
+Example for your services:
+```yaml
+# .platform/services.yaml
+database:
+  type: postgresql:15
+  disk: 1024
+```
+
+{{< note >}}
+These settings, size, disk and resources are no longer available in the Upsun Yaml config files (or not taken in account anymore).
+{{< /note >}}
+
+### Using the API
+#### Change app resources
+To change your `app` application resources, use the following command line:
+```
+$ upsun e:curl /deployments/next -X PATCH -d \
+  '{
+    "webapps": {
+    "app": {
+      "resources": {"profile_size": "2"},
+      "disk": "1024"
+    }
+  }
+}'
+```
+
+{{< note >}}
+Don’t forget to remove from your `.platform/config.yaml` file corresponding Yaml key that are not needed anymore: `size`, `disk` and `resources`
+{{< /note >}}
+
+#### Change database service resources
+To change database service resources, use the following command line:
+```shell
+$ upsun e:curl /deployments/next -X PATCH -d \
+  '{
+    "services": {
+    "database": {
+      "disk": "1024",
+      "resources": {"profile_size": "1"}
+    }
+  }
+}'
+```
+
+{{< note >}}
+Don’t forget to remove from your .platform/services.yaml file corresponding Yaml key that are not needed anymore: `disk`
+{{< /note >}}
+
+[//]: # (TODO change it to doc page link)
+List of profile_size can be founded [here](https://docs.google.com/presentation/d/1PxDJm9c3OCjZxlI2keGy6V7BlAZJGfyueQ1D117YCns/edit?pli=1#slide=id.g20fc593a519_0_32)
+
+### Using CLI (TODO)
+TODO
+### Using Console (TODO)
+TODO
+
+## Deploy your Upsun project
+After adding to Git your changes, use the following command to deploy your Upsun project:
+```shell
+upsun deploy
```

Comment:
are we sure the upsun cli has a `deploy` command? Latest version (4.2.1, Build date: 2023-08-09T10:22:55+00:00) does not contain it.